compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/node/NodeCoordinator.kt

```diff
@@ -1449,7 +1449,6 @@ internal abstract class NodeCoordinator(
         )
     }
 
-    @kotlin.native.concurrent.ThreadLocal
```

Comment:
Quite suspicious. What the context? Is runtime behavior affected?

Reply:
I can't speak for sure, but:
- it was added during the initial iOS support
- that time we had issues with the native old native memory model, where we couldn't share a mutable value across different threads
- as far as I checked, there is no multi-threading in case of a single `ComposeScene`. In case of multiple `ComposeScene`, we should add a Thread local for all targets, including Android, and not only here, but for all global objects.